cypress/fixtures/credentials.json

```diff
@@ -1,4 +1,6 @@
 {
-  "email" : "loripsum@yahoo.com",
-  "password" : "qazwsx123"
-}
\ No newline at end of file
+  "emailValid": "loripsum@yahoo.com",
+  "emailUnregistered": "loripsum+1@yahoo.com",
+  "passwordValid": "qazwsx123",
+  "passwordInvalid": "qazwsx1234"
```

Comment:
please do not put yourself in limited conditions by storing invalid values if it is not something specific
it would be better if you use some kind of generators for this
we don't really care about the content of invalid password, so please try to generate it on the go with `chance` library for example

Reply:
we have discussed it on the call and decided to leave as it is